src/pytest_regressions/dataframe_regression.py

```diff
@@ -1,5 +1,14 @@
-from pytest_regressions.common import import_error_message
-from pytest_regressions.common import perform_regression_check
+# mypy: disallow-untyped-defs
```

Comment:
Could this config (repeated in many files) be moved to `mypy.ini`?

Reply:
Good idea, done!